src/main/java/org/jenkinsci/plugins/scriptsecurity/scripts/ScriptApproval.java

```diff
@@ -668,8 +668,14 @@ public synchronized RejectedAccessException accessRejected(@Nonnull RejectedAcce
     @Restricted(NoExternalUse.class) // implementation
     @Extension public static final class ApprovedWhitelist extends ProxyWhitelist {
         public ApprovedWhitelist() throws IOException {
-            reconfigure();
+            try {
+                reconfigure();
+            } catch (IOException e) {
+                LOG.log(Level.SEVERE, "Malformed signature entry in scriptApproval.xml: '" + e.getMessage() + "'");
+                throw e;
```

Comment:
Why even rethrow?

It is not clear from either JIRA or the PR who was catching the `IOException` to begin with and where it was swallowed, but anyway I see no reason to rethrow this. Just delete and remove the `throws` clause.